modules/assim.sequential/inst/NEFI/README.md

```diff
@@ -0,0 +1,28 @@
+Steps for running and adding SIPNET forecasts 
+==========================================================================
+ 
+### Files associated with running forecasts:
+1. `generate.gefs.xml` : R script for setting start and end date in xml; creates workflow id and associated directory
+2. `*sitename*.xml` : site specific xml script 
+3. `*sitename*.sh` : site specific shell file to run forecasts with cron jobs 
+
+### Files associated with graphing forecasts: 
+1. `download_*sitename*.R` : R script for downloading and cleaning Flux data from towers 
+2. `graph_fluxtowers.R` : script to create RData for shiny app 
+3. `forecast.graphs.R` : script that is used to get forecasted data for `graph_fluxtowers.R`
+4. `graph_SDA_fluxtowers.R` : script to graph SDA fluxtowers 
+5. `automatic_graphs.sh` : shell for running `email.R`, `graph_fluxtowers.R`, `graph_SDA_fluxtowers.R`
+
+### Files for automatic WCr emails: 
+1. `email_graphs.R` - creates graphs for the email - this is specifically for WCr right now
+2. `email.R` - sends the email out with graphs for today's forecast
+
+
+
+
+### To add forecast and add to dashboard: 
+1. create a flux download for site
+2. create site specific xml 
+3. create site specific shell file to run forecast automatically 
```

Comment:
conceptual question: Once you have a site-specific download function and settings xml, why would the shell script to run the forecast need to be site specific?

Reply:
The shell script is site specific right now because it is points to the site's xml file and output path. The sh file runs the R script to update the xml file with the new dates and workflow ids and then runs the pecan workflow with the updated xml file. The site specific flux download function is for graphing since that's when we compare the forecast to the actual data.

Reply:
Will pull at this point, but I'm unconvinced that we need site-specific scripts -- based on your description it sounds like you could have a single shell script that takes the xml and outpath as arguments

If the download function is for graphing, where do you download the data for assimilation?

Reply:
Oh yeah, I could do a shell with those as xml and outpath. That would be good for when we scale up. As for the download, these aren't assimilating data right now, they are just the regular SIPNET forecasts but it's the same function for downloading for data assimilation